new/LIT7086QeneBoSadqan.xml

```diff
@@ -0,0 +1,90 @@
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Qǝne</foreign> without type indication in the manuscript. It has nine lines, meaning
+                    that it can be classified as a <foreign xml:lang="gez">mawaddəs qəne</foreign> 
+                    (possibly of the subtype <foreign xml:lang="gez">kʷəlləkəmu</foreign>) according to
+                    <bibl><ptr target="bm:Guidi1900Qene"/></bibl>. 
+                    It is attributed to <persName ref="PRS14574GabraLeul" cert="high" corresp="SD">Gabra Ləʿul</persName> 
+                    in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Mawaddes" cert="medium"/>
+                    <term key="Kwellekomu" cert="medium"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-10-04">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7086QeneBoSadqan" passive="UppQ1524o"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7086QeneBoSadqan" passive="PRS14574GabraLeul"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="UppQ1524o"/> on fols 18r, 19r. Division into verses is indicated with ፤ .
+                    The end of the poem is indicated with ። .</note>
+                <ab>
+                    <l n="1">ቦ፡ ጻድቃን፡ ውሉ<del rend="strikethrough">ድ</del><add place="above">ደ</add>፡ 
+                        ቤትከ፡ እለ፡ መነኑ፡ ዓለ<del rend="strikethrough">ም</del><add place="above">መ</add>፡ ወዕለ፡ ኤሉ፡ አድባረ፤</l>
+                    <l n="2">እንዘ፡ ይሐስሱ፡ ዮም፡ ሰማያዊተ፡ ሀገ<del rend="strikethrough">ር</del><add place="above">ረ</add>፤</l>
+                    <l n="3">እስከነ፡ ቀነዩ፡ ሥጋሆሙ፡ ገብረ፤</l>
+                    <l n="4">ወልቡናሆሙ፡ ማዕምረ፤</l>
+                    <l n="5">ለተቀንዎ፡ አምላክ፡ ዘኢያህጸርከ፡ አእጋረ፤</l><pb n="19r"/>
+                    <l n="6">ንህነሂ፡ እለ፡ ጸሐፍነ፡ ሐኬተ፡ ልቡና፡ <choice resp="JK"><sic>ጠ</sic><corr>ጦ</corr></choice>ማረ፤</l>
+                    <l n="7">ወአምረርነ፡ ቃላቲከ፡ ሐሊበ፡ ወመአረ፤</l>
+                    <l n="8">እንዘ፡ ተድላ፡ ዓለም፡ ንሐስስ፡ ወንትሜነዶ፡ ክብረ፤</l>
```

Comment:
The ዓ in ዓለም፡ looks very much like ፃ. You may decide, if you want to keep it like this or mark (your) correction to ዓለም፡.

Reply:
OK. Perhaps the thin stroke inside ዓ was unintentionally.

Reply:
I would say that it probably is an error, but I don't know how to markup this kind of ambiguous characters

Reply:
What you can do, is to explain the case explicitly in the abstract, but I do not think, that this case is worth being discussed in the abstract.

Reply:
Yes, I agree that it goes a bit too much into detail. Maybe I can remark on this and other similar phenomena in the hand description in the manuscript record (if I remember)